droidlet/tools/hitl/utils/allocate_instances.py

```diff
@@ -320,7 +324,7 @@ def free_ecs_instances(instance_ids):
         "--task_name", type=str, help="Task name of the ecs instance to be requested"
     )
     args = parser.parse_args()
-    instance_ips, instance_ids = request_instance(args.instance_num, args.image_tag, args.task_name)
+    instance_ips, instance_ids = request_instance(args.instance_num, args.image_tag, args.task_name, batch_id="0643")
```

Comment:
why is the batch id hard coded here?

Reply:
Everything under __main__() is test code. I tested with this dummy batch_id to show it works.